libraries/SITL/SITL.cpp

```diff
@@ -563,37 +610,97 @@ const AP_Param::GroupInfo SIM::var_sfml_joystick[] = {
+    // @Description: Temperature simulation parameter. Valid only if IMUT_FIXED=0
     AP_GROUPINFO("IMUT_START",    1, SIM, imu_temp_start,  25),
+    // @Param: IMUT_END
+    // @DisplayName: IMU Temp End
+    // @Description: Temperature simulation parameter. Valid only if IMUT_FIXED=0
     AP_GROUPINFO("IMUT_END",      2, SIM, imu_temp_end, 45),
+    // @Param: IMUT_TCONST
+    // @DisplayName: IMU Temp Constant
+    // @Description: Temperature simulation parameter. Valid only if IMUT_FIXED=0
     AP_GROUPINFO("IMUT_TCONST",   3, SIM, imu_temp_tconst, 300),
+    // @Param: IMUT_FIXED
+    // @DisplayName: IMU Temp Fixed
+    // @Description: Fixed temperature simulation value. If 0, temperature simulation is calculated from the current time with parameters IMUT_START, IMUT_END, IMUT_TCONST.
     AP_GROUPINFO("IMUT_FIXED",    4, SIM, imu_temp_fixed, 0),
 #endif
+    // @Param: ACC1_BIAS
+    // @DisplayName: ACCEL1 Bias
+    // @Description: Simulated Bias of ACCEL1
+    // @Units: m/s/s
     AP_GROUPINFO("ACC1_BIAS",     5, SIM, accel_bias[0], 0),
 #if INS_MAX_INSTANCES > 1
+    // @Param: ACC2_BIAS
+    // @DisplayName: ACCEL2 Bias
+    // @Description: Simulated Bias of ACCEL2
+    // @Units: m/s/s
     AP_GROUPINFO("ACC2_BIAS",     6, SIM, accel_bias[1], 0),
 #endif
 #if INS_MAX_INSTANCES > 2
+    // @Param: ACC3_BIAS
+    // @DisplayName: ACCEL3 Bias
+    // @Description: Simulated Bias of ACCEL3
+    // @Units: m/s/s
     AP_GROUPINFO("ACC3_BIAS",     7, SIM, accel_bias[2], 0),
 #endif
+    // @Param: GYR1_RND
+    // @DisplayName: GYR1 Random
+    // @Description: Simulated Noise of Gyroscope 1
+    // @Units: degrees/second
     AP_GROUPINFO("GYR1_RND",      8, SIM, gyro_noise[0],  0),
 #if INS_MAX_INSTANCES > 1
+    // @Param: GYR2_RND
+    // @DisplayName: GYR2 Random
+    // @Description: Simulated Noise of Gyroscope 2
+    // @Units: degrees/second
     AP_GROUPINFO("GYR2_RND",      9, SIM, gyro_noise[1],  0),
 #endif
 #if INS_MAX_INSTANCES > 2
+    // @Param: GYR3_RND
+    // @DisplayName: GYR3 Random
+    // @Description: Simulated Noise of Gyroscope 3
+    // @Units: degrees/second
     AP_GROUPINFO("GYR3_RND",     10, SIM, gyro_noise[2],  0),
 #endif
+    // @Param: ACC1_RND
+    // @DisplayName: ACC1 Random
+    // @Description: Simulated Noise of ACCEL1
+    // @Units: m/s/s
     AP_GROUPINFO("ACC1_RND",     11, SIM, accel_noise[0], 0),
 #if INS_MAX_INSTANCES > 1
+    // @Param: ACC2_RND
+    // @DisplayName: ACC2 Random
+    // @Description: Simulated Noise of ACCEL2
+    // @Units: m/s/s
     AP_GROUPINFO("ACC2_RND",     12, SIM, accel_noise[1], 0),
 #endif
 #if INS_MAX_INSTANCES > 2
+    // @Param: ACC2_RND
+    // @DisplayName: ACC2 Random
+    // @Description: Simulated Noise of ACCEL2
+    // @Units: m/s/s
     AP_GROUPINFO("ACC3_RND",     13, SIM, accel_noise[2], 0),
 #endif
+    // @Param: GYR1_SCALE
+    // @DisplayName: GYR1 Scale
+    // @Description: GYR1 scale factor
+    // @Units: %
```

Comment:
Is this *really* a percent?

Reply:
You are right. My mistake, the gyroscope scale is measured in rad/sec. I'll also change the misleading comment on https://github.com/ArduPilot/ardupilot/blob/b638a3576b1a1d397933891c6512c4a11e990f95/libraries/SITL/SITL.h#L496

Reply:
Mmmmm... I think you will find these need to be unit-less for it to be correct units-wise.

If the scale was radians then you'd have radians-per-second/radians-per-second which would mean the result would be unit-less - but we know that the gyro data has to be in radians-per-second....

Scales are usually unit-less.

I've just had a look - turns out I'm wrong, these *are* percentages!

https://github.com/ardupilot/ardupilot/blob//libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp#L283

Apologies for the run-around.  If you could add a comment on that line 496 saying, "look in AP_InertialSensor_SITL for details" that would be good.

My bad.